frontend/tests/e2e/tests/sfdsAuthorizations.js

```diff
@@ -0,0 +1,67 @@
+import { itsbAccessTeamUser } from "../roles/roles"
+import { Selector } from "testcafe";
+
+const SFDS_CLIENT = 'SFDS';
+const TEST_CAFE_USER_ID = '3195a1bf-4bea-47c4-955d-cf52d4e2fc15';
+const SFDS_MAILBOX_TO_ADD = 'BCMA';
```

Comment:
I think we store all mailbox_id's as lowercase because their corresponding physical mailbox directories are all lowercase...not sure if that even matters for this specific test.